packages/ketcher-core/src/domain/entities/DrawingEntitiesManager.ts

```diff
@@ -175,6 +175,33 @@ export class DrawingEntitiesManager {
     return command;
   }
 
+  private addOperationToCommand(
+    command: Command,
+    drawingEntity: DrawingEntity,
+  ) {
+    const operation = new DrawingEntitySelectOperation(drawingEntity);
+    command.addOperation(operation);
+  }
+
+  public selectCurrentDrawingEntities(drawingEntity: DrawingEntity) {
+    const command = new Command();
+
+    this.allEntities.forEach(([, drawingEntity]) => {
+      if (drawingEntity.selected) {
+        this.addOperationToCommand(command, drawingEntity);
+      }
+    });
+
+    if (drawingEntity.selected) {
+      drawingEntity.turnOffSelection();
+    } else {
+      drawingEntity.turnOnSelection();
+    }
+    this.addOperationToCommand(command, drawingEntity);
+
+    return command;
+  }
```

Comment:
It looks like a bit overhead to create operations for already selected entities. Can we rework and rename this method to add only one drawing entity to selection?